test/Dispatcher.t.sol

```diff
@@ -4,23 +4,18 @@ pragma solidity ^0.8.13;
 import "../contracts/libs/Ibc.sol";
 import {Dispatcher} from "../contracts/core/Dispatcher.sol";
 import {IbcEventsEmitter} from "../contracts/interfaces/IbcDispatcher.sol";
-import {IbcReceiver, IbcReceiverBase} from "../contracts/interfaces/IbcReceiver.sol";
+import {IbcReceiver} from "../contracts/interfaces/IbcReceiver.sol";
+import {DummyLightClient} from "../contracts/utils/DummyLightClient.sol";
+import {TestUtils} from "./TestUtils.sol";
 import "../contracts/examples/Mars.sol";
-import "../contracts/core/OpConsensusStateManager.sol";
+import "../contracts/core/OpLightClient.sol";
 import "./Dispatcher.base.t.sol";
 
-contract ChannelHandshakeTest is Base {
+abstract contract ChannelHandshakeTestSuite is Base {
```

Comment:
why do we new the `abstract` keyword here? are we missing implementation of any method?

Reply:
[This](https://book.getfoundry.sh/forge/writing-tests#shared-setups) was the pattern I'm following; not sure exactly why they mark it as abstract; might be so that the inheriting contracts only run the setup once, or might have something to do with the test output